### ISSUE TITLE
Code Highlight not working as docs

### ISSUE THREAD

**Issue by alebeta90** [mattermost/mattermost-server#13068]
Hello friends,

#### Summary
Writing code blocks is not highlighting the code properly as it is shown in the documentation

Documentation:

![image](https://user-images.githubusercontent.com/11571266/68698072-6cf26880-0580-11ea-8245-b0140428f654.png)

In My Mattermost Server:

![image](https://user-images.githubusercontent.com/11571266/68698114-80053880-0580-11ea-8354-6cc0e2e746b1.png)

There are some difference in how the text inside is more readable using different colours.

#### Steps to reproduce

I did a docker deployment and it is running version 5.16.3

#### Expected behavior
shows code as docs

#### Observed behavior (that appears unintentional) 
it does not shows as docs

thanks

**Comment by amyblais** [comment]
@alebeta90 Please help share the full code that you used as well as the documentation that you followed.

**Comment by alebeta90** [comment]
Hi @amyblais ,

sorry for the lack of information

The documentation is this one
https://docs.mattermost.com/help/messaging/formatting-text.html

and the code I used was

``` go
``` go
package main
import "fmt"
func main() {
    fmt.Println("Hello, 世界")
}
```

I can not add the full snipe code because mark down recognized.

that it is the same in the documentation

If there is more information I could bring, please let me know

**Comment by amyblais** [comment]
@alebeta90 We have this in-progress bug ticket open that is probably related to the issue you're seeing: https://mattermost.atlassian.net/browse/MM-19964.

**Comment by alebeta90** [comment]
@amyblais thanks for the information,

I saw there is already a PR to fix this issue that is waiting for review.

Than is just a matter of wait :D 

thanks a lot

**Comment by amyblais** [comment]
Closing as this was fixed for v5.17,
